app/routes.py

```diff
@@ -1,2 +1,239 @@
-from flask import Blueprint
+from flask import Blueprint, json, jsonify, request, make_response
+from werkzeug.wrappers import response
+from app.models.task import Task
+from app.models.goal import Goal
+from app import db
+import requests
+from datetime import datetime
+import os
+from dotenv import load_dotenv
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+@tasks_bp.route("", methods=["POST"])
+def post_tasks():
+    request_body = request.get_json()
+    if "title" not in request_body or "description" not in request_body or "completed_at" not in request_body:
+        return make_response({"details":"Invalid data"}, 400)
```

Comment:
I tend to prefer using jsonify (as you did on line 32) over make_response (as you did here). For me, it helps make it clearer what we're expecting to return, and if we removed all the `make_response` calls, we could stop importing it.